qa/suites/upgrade/jewel-x/point-to-point-x/point-to-point-upgrade.yaml

```diff
@@ -202,13 +205,7 @@ upgrade-sequence_x:
    - ceph.restart: [osd.4]
    - sleep:
        duration: 30
-   - ceph.restart:
-       daemons: [osd.5]
-       wait-for-healthy: false
-       wait-for-up-osds: true
-   - exec:
-       mon.a:
-         - ceph osd set require_jewel_osds
```

Comment:
This deleted block needs to be added at the end of `upgrade-sequence_jewel` so it runs when the final daemon (`mon.c`) is upgraded to v10.2.7.